### ISSUE TITLE
Workflow with DOI can't have title or information text changed

### ISSUE THREAD

**Issue by douglowe** [seek4science/seek#589]
**Describe the bug**
I had changed the title and information text of this workflow [https://workflowhub.eu/workflows/56](https://workflowhub.eu/workflows/56) for version 1. When I created version 2 it looked like these changes had been carried forward, but creating a DOI for version 2 (I hadn't created a DOI for version 1) caused this information to revert to the original information.

The title for version 1 and 2 is the same (old) title. The information text for version 1 is the updated information text (which was updated before I created version 2), but the information text for version 2 (which I'd not modified when I created the new version, I let WFH set this from the previous versions information) is the old information text.

Now I can't change the title or information text for the workflow. I get a 'Un-publishing this asset is not possible' error. Is this correct (I guess, perhaps, if the workflow has been stored on datacite, then perhaps we should be able to change the entry anymore)? And why did the text and title change in either the new upload or the minting of the DOI?

**Comment by johannes-darms** [comment]
@whomingbird : Do we have the same issue with other asset types?

**Comment by whomingbird** [comment]
yes, i also spotted this bug when i was checking DOI feature, but didn't know there is already an issue.
